scripts/zones/Ceizak_Battlegrounds/mobs/Taxet.lua

```diff
@@ -0,0 +1,80 @@
+-----------------------------------
+-- Area: Ceizak Battlegrounds
+-- NM: Tax'et
+-----------------------------------
+local entity = {}
+
+local removables = {
+    xi.effect.FLASH, xi.effect.BLINDNESS, xi.effect.ELEGY, xi.effect.REQUIEM, xi.effect.PARALYSIS, xi.effect.POISON,
+    xi.effect.CURSE_I, xi.effect.CURSE_II, xi.effect.DISEASE, xi.effect.PLAGUE, xi.effect.WEIGHT, xi.effect.BIND,
+    xi.effect.BIO, xi.effect.DIA, xi.effect.BURN, xi.effect.FROST, xi.effect.CHOKE, xi.effect.RASP, xi.effect.SHOCK, xi.effect.DROWN,
+    xi.effect.STR_DOWN, xi.effect.DEX_DOWN, xi.effect.VIT_DOWN, xi.effect.AGI_DOWN, xi.effect.INT_DOWN, xi.effect.MND_DOWN,
+    xi.effect.CHR_DOWN, xi.effect.ADDLE, xi.effect.SLOW, xi.effect.HELIX, xi.effect.ACCURACY_DOWN, xi.effect.ATTACK_DOWN,
+    xi.effect.EVASION_DOWN, xi.effect.DEFENSE_DOWN, xi.effect.MAGIC_ACC_DOWN, xi.effect.MAGIC_ATK_DOWN, xi.effect.MAGIC_EVASION_DOWN,
+    xi.effect.MAGIC_DEF_DOWN, xi.effect.MAX_TP_DOWN, xi.effect.MAX_MP_DOWN, xi.effect.MAX_HP_DOWN
+}
+
+local exuviationSkillID = 1955
+local initialDamage = -9900
+local damageIncrease = 494
+local exuviationCD = 10
+
+local function hasNegativeEffect(mob)
+    for _, effect in ipairs(removables) do
+        if mob:hasStatusEffect(effect) then
+            return true
+        end
+    end
+    return false
+end
+
+local function updateDamageReduction(mob)
+    local exuviationUsageCount = mob:getLocalVar('exuviationUsageCount') or 0
+    local reduction = initialDamage + (exuviationUsageCount * damageIncrease)
+    reduction = math.min(reduction, 0)
+
+    mob:setMod(xi.mod.UDMGPHYS, reduction)
+    mob:setMod(xi.mod.UDMGMAGIC, reduction)
+    mob:setMod(xi.mod.UDMGRANGE, reduction)
+    mob:setMod(xi.mod.UDMGBREATH, reduction)
+end
+
+local function handleExuviationUsage(mob)
+    local exuviationUsageCount = mob:getLocalVar('exuviationUsageCount') or 0
+    exuviationUsageCount = exuviationUsageCount + 1
+    mob:setLocalVar('exuviationUsageCount', exuviationUsageCount)
```

Comment:
To reduce the chance of these localvars being typo'd, can you extract `'exuviationUsageCount'` into a local table:
```lua
local vars = 
{
    EXUVIATION_USAGE_COUNT = 'exuviationUsageCount',
}

...

 local exuviationUsageCount = mob:getLocalVar(vars.EXUVIATION_USAGE_COUNT) or 0
```